### PR TITLE
improve the sample app list styling

### PR DESCRIPTION
**Pull request by ts-nitish-a-jain**
# Description
Improve the list styling and make it look better while balancing the white space.

From:
image

To:
image

## Links
[MINI-1220](https://jira.rakuten-it.com/jira/browse/MINI-1220)

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: testapp/src/main/res/layout/item_section_miniapp.xml [hunk 1]
```diff
@@ -2,44 +2,36 @@
...
             android:id="@+id/tv_app_name"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:ellipsize="end"
+            android:padding="@dimen/small_5"
             android:singleLine="true"
             android:text="@{miniapp.displayName}"
             android:textColor="@android:color/black"
-            android:textSize="@dimen/text_large_16"
-            android:padding="@dimen/medium_16"
```
Comment (corycaywood) [inline testapp/src/main/res/layout/item_section_miniapp.xml:38]: I think that this still needs `dimen/medium_16` for left and right padding. The text looks too close to the edge of the screen.

Comment (ts-nitish-a-jain) [reply testapp/src/main/res/layout/item_section_miniapp.xml:38]: Okay, that's manageable by just the paddingStart/End. Modified.

**Comment by ts-nitish-a-jain** [comment]
> I think the original one is better.
> I don't have strong opinion about it but you can ask our team or QA guys about it.

For UI/UX decisions, when it is needed, if there's consensus within the team, it is good. 
Otherwise, it needs to be coming from the Product.

Also, when changing UI, please put in screenshots from before & after so that there is less chance of a skip.

File: testapp/src/main/res/layout/item_section_miniapp.xml (hunk 1 above)
Comment (ts-nitish-a-jain) [reply testapp/src/main/res/layout/item_section_miniapp.xml:38]: I have currently reset it to 8dp as it was giving a distinct factor, 16 was making it quite in line. 
Put all the 4 screenshots for reference to give a comparative view.

image
image
image
image

Let me know @corycaywood @minh-rakuten

Comment (minh-rakuten) [reply testapp/src/main/res/layout/item_section_miniapp.xml:38]: 4️⃣ is good for me.

Comment (corycaywood) [reply testapp/src/main/res/layout/item_section_miniapp.xml:38]: I agree that 4 looks the best. 3 looks mis-aligned.

Comment (ts-nitish-a-jain) [reply testapp/src/main/res/layout/item_section_miniapp.xml:38]: Lol. Okay. 4th it is then.

**Pull request merged**